lib/modules/dosomething/dosomething_campaign/includes/CampaignTransformer.php

```diff
@@ -83,7 +83,7 @@ public function show($id) {
    * @return array
    */
   protected function transform($item) {
-    $data = array();
+    $data = [];
```

Comment:
I read in the Drupal forums that `array()` is dead and everything is moving to `[ ]`. Is this the beginning?  :)